### PR TITLE
Add missing webpackChunkName magic comments

### PR DESCRIPTION
**Pull request by billfienberg**
## Description

Add/update `webpackChunkName` [magic comments](https://webpack.js.org/api/module-methods/#magic-comments)

### More context

Webpack supports "magic comments."

>Inline comments to make features work. By adding comments to the import, we can do things such as name our chunk or select different modes.  - https://webpack.js.org/api/module-methods/#magic-comments

While inspecting the webpack build, I noticed there were some bundle files that had unintuitive names, like (`0.entry.js`, `1.entry.js`, `2.entry.js`, `3.entry.js`). We already use the `webpackChunkName` magic comment for some dynamic imports, and this PR adds/updates the `webpackChunkName` to some dynamic imports. 

## Testing done

CI

## Links

- https://dsva.slack.com/archives/CQH357ZTP/p1600876610006000
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/8983

## Acceptance criteria
- [x] CI passes
- [ ] bundle filenames are updated from unintuitive names like `1.entry.js` to `dashboard.entry.js`

## Definition of done
- [ ] Documentation has been updated, if applicable

### CODE REVIEW

**Review by jhonny218 — APPROVED** [review]
LGTM

**Comment by alexandec** [comment]
FYI I just noticed this note on more readable automatic chunk names in webpack 5: https://webpack.js.org/blog/2020-10-10-webpack-5-release/#major-changes-development-support. I haven't tried it yet though

**Review by ncksllvn — APPROVED** [review]

**Comment by billfienberg** [comment]
> FYI I just noticed this note on more readable automatic chunk names in webpack 5: https://webpack.js.org/blog/2020-10-10-webpack-5-release/#major-changes-development-support. I haven't tried it yet though

@alexandec, that's cool. Thanks for pointing that out. 

>So you no longer need to use import(/* webpackChunkName: "name" */ "module") for debugging. But it would still make sense if you want to control the filenames for production environments. - https://webpack.js.org/blog/2020-10-10-webpack-5-release/#major-changes-development-support

**Comment by alexandec** [comment]
@billfienberg good point, looks like the new automatic naming is for dev only. It's promising but still limited

**Review by mdewey — APPROVED** [review]
LGTM!

**Review by jbalboni — APPROVED** [review]

**Review by LeKeve — APPROVED** [review]

File: src/applications/third-party-app-directory/createThirdPartyApps.js [hunk 1]
```diff
@@ -15,14 +15,15 @@ export default (store, widgetType) => {
...
-  import('./third-party-apps-entry').then(module => {
-    const { App } = module.default;
-    ReactDOM.render(
-      <Provider store={store}>
-        <App />
-      </Provider>,
-      root,
-    );
-  });
+  import(/* webpackChunkName: "third-party-app-directory" */ './third-party-apps-entry').then(
```
Comment (U-DON) [inline src/applications/third-party-app-directory/createThirdPartyApps.js:18]:
```suggestion
  import(/* webpackChunkName: "third-party-apps-directory" */ './third-party-apps-entry').then(
```

**Pull request merged**